mmdet/apis/det_inferencer.py

```diff
@@ -339,7 +339,8 @@ def __call__(self,
             ori_inputs, batch_size=batch_size, **preprocess_kwargs)
 
         results_dict = {'predictions': [], 'visualization': []}
-        for ori_inputs, data in track(inputs, description='Inference'):
+        for ori_inputs, data in (track(inputs, description='Inference')
```

Comment:
You need to write the show_progress parameter to the det_inferencer initialization list, otherwise it cannot be modified

Reply:
ok, i was setting it after creation. but better to allow passing to `__init__ `. will update. thx!